date_range/__manifest__.py

```diff
@@ -1,9 +1,9 @@
-# © 2016 ACSONE SA/NV (<http://acsone.eu>)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# © 2016 ACSONE SA/NV (<https://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Date Range",
     "summary": "Manage all kind of date range",
-    "version": "11.0.2.0.1",
+    "version": "11.0.3.0.1",
```

Comment:
@ntsirintanis Can you also update the history as in https://github.com/OCA/server-tools/commit/ed05e386ac237f5b8f0344e5f6e7088609aae639#diff-3db8a615cdc543d077b5b3cfbc7f939d